explorer/README.ETCExplorer.md

```diff
@@ -0,0 +1,82 @@
+# ETCExplorer
```

Comment:
Replace ETC with Kowala references & remove extra links.

Reply:
or remove file?

Reply:
not sure, we should keep the original. I added our own README